config/locales/devise.nl.yml

```diff
@@ -0,0 +1,63 @@
+# if me: DO NOT modify this file. if you need to add translations, add them to en.yml
+# Additional translations at https://github.com/plataformatec/devise/wiki/I18n
+
+nl:
+  devise:
+    confirmations:
+      confirmed: "Je emailadres is met succes bevestigd."
+      send_instructions: "Je zult een email ontvangen ter bevestiging en verdere instructies binnen enkele minuten."
```

Comment:
This feels a little clunky. I'd change this to:

"Je zult binnen enkele minuten een email ontvangen met instructies om je email adres te bevestigen."